lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -47,6 +50,27 @@ class PrimaryNewsListItem extends StatelessWidget {
           bottom: 0,
           child: _buildNewsInfo(context),
         ),
+        Positioned.fill(
+          child: ClipRRect(
+            borderRadius: BorderRadius.circular(_Constants.imageRadius),
+            child: Opacity(
+              opacity: _Constants.newsImageLayerOpacity,
```

Comment:
skoro jest transparent to po co Opacity?

Reply:
ripple effect jest nieco bardziej delikatny ;)

Reply:
A to nie zauważyłem :D